packages/edit-site/src/components/block-editor/resizable-editor.js

```diff
@@ -182,7 +182,10 @@ function ResizableEditor( { enableResizing, settings, ...props } ) {
 				name="editor-canvas"
 				className="edit-site-visual-editor__editor-canvas"
 				{ ...props }
-			/>
+			>
+				{ settings.svgFilters }
```

Comment:
Filters need to be rendered before children to avoid Safari rendering issues.

Reply:
Can we enqueue the default and theme SVGs from the server instead? The only change in the client would be to create the selectors using the duotone selector.

Reply:
I reverted this change because I expect we'll need this for updating the duotone filters in the site editor eventually

Reply:
> Filters need to be rendered before children to avoid Safari rendering issues.

Can we add this as a comment in the code to document why is was done this way?